wsmaster/che-core-api-factory-bitbucket-server/pom.xml

```diff
@@ -70,15 +94,28 @@
             <groupId>org.eclipse.che.core</groupId>
             <artifactId>che-core-commons-annotations</artifactId>
         </dependency>
+        <dependency>
+            <groupId>org.eclipse.che.core</groupId>
+            <artifactId>che-core-commons-inject</artifactId>
+        </dependency>
         <dependency>
             <groupId>org.eclipse.che.core</groupId>
             <artifactId>che-core-commons-lang</artifactId>
         </dependency>
+        <dependency>
+            <groupId>org.slf4j</groupId>
+            <artifactId>slf4j-api</artifactId>
+        </dependency>
         <dependency>
             <groupId>ch.qos.logback</groupId>
             <artifactId>logback-classic</artifactId>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>com.github.tomakehurst</groupId>
+            <artifactId>wiremock-jre8-standalone</artifactId>
```

Comment:
Is this correct when we're on Java 11?

Reply:
AFAIK the reason to use wiremock-jre8-standalone is the conflict of Jetty that is used in wiremock and we are using in everrest assured. To workaround that we are using all-in-one-bundle. Wich has no java11 equivalent.